notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>This is just a comment, not a code change request. We usually split the imports into global/specific imports. For example: sys, time, os, etc are all standard Python libraries. So they are imported first. Then we import libraries that need to be installed but they are popular, like numpy and matplotlib. Finally, we have imports that are specific to our project. Basically everything that is associated with last. Here is more or less how I would put them:</p><pre class="ql-syntax" spellcheck="false">import asyncio
import glob
import os
import shlex
import subprocess
import sys
from datetime import datetime

import matplotlib.pyplot as plt
import numpy as np
from astropy.time import Time, TimeDelta
from matplotlib.colors import LightSource

import lsst.ts.cRIOpy.M1M3FATable as M1M3FATable
from lsst_efd_client import EfdClient
</pre><p>There is probably a standard on this. But I don't have the documentation with me now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done